pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -1509,7 +1509,7 @@ func runTPCCBench(ctx context.Context, t test.Test, c cluster.Cluster, b tpccBen
 	// warehouses that can be operated on while sustaining a throughput
 	// threshold, set to a fraction of max tpmC.
 	precision := int(math.Max(1.0, float64(b.LoadWarehouses(c.Cloud())/200)))
-	initStepSize := precision
+	initStepSize := precision * 2
```

Comment:
Increasing the step size tends to reduce the total number of line search iterations. E.g., two runs after the change,

```
grep -E "SEARCH ITER|MAX|success" artifacts/tpccbench/nodes\=9/cpu\=4/multi-region/run_1/test.log

00:55:09 tpcc.go:1678: --- SEARCH ITER FAIL: TPCC 2600 resulted in 19881.2 tpmC and failed due to efficiency value of 60.663357109930885 is below passing threshold of 85
01:14:14 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2570 resulted in 31678.6 tpmC (97.8% of max tpmC)
01:33:18 tpcc.go:1678: --- SEARCH ITER FAIL: TPCC 2585 resulted in 22991.8 tpmC and failed due to efficiency value of 70.56187811080218 is below passing threshold of 85
MAX WAREHOUSES = 2577
01:33:55 test_runner.go:1159: test completed successfully

03:32:24 tpcc.go:1678: --- SEARCH ITER FAIL: TPCC 2600 resulted in 27032.3 tpmC and failed due to efficiency value of 82.48333885126067 is below passing threshold of 85
03:51:31 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2570 resulted in 31911.8 tpmC (98.5% of max tpmC)
04:10:37 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2585 resulted in 32029.4 tpmC (98.3% of max tpmC)
MAX WAREHOUSES = 2592
04:11:12 test_runner.go:1159: test completed successfully
```

vs. before the change,

```
16:45:45 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2000 resulted in 25079.7 tpmC (99.5% of max tpmC)
17:04:47 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2015 resulted in 25286.7 tpmC (99.6% of max tpmC)
17:23:54 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2045 resulted in 25659.2 tpmC (99.5% of max tpmC)
17:43:00 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2105 resulted in 26349.3 tpmC (99.3% of max tpmC)
18:02:02 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2225 resulted in 27857.7 tpmC (99.3% of max tpmC)
18:21:07 tpcc.go:1678: --- SEARCH ITER FAIL: TPCC 2465 resulted in 23005.4 tpmC and failed due to efficiency value of 74.04061603766169 is below passing threshold of 85
18:40:13 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2345 resulted in 29223.6 tpmC (98.9% of max tpmC)
18:59:21 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2405 resulted in 29923.2 tpmC (98.7% of max tpmC)
19:18:30 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2435 resulted in 26741.5 tpmC (87.1% of max tpmC)
19:37:37 tpcc.go:1674: --- SEARCH ITER PASS: TPCC 2450 resulted in 27288.9 tpmC (88.4% of max tpmC)
MAX WAREHOUSES = 2457
19:38:14 test_runner.go:1159: test completed successfully
```